ROADMAP.md

```diff
@@ -0,0 +1,45 @@
+# **librpma roadmap [WIP]**
+
+This is the roadmap of librpma library. It consists of high-level features that are going to be released with the library release 1.0.
+This roadmap is for informational purposes only, so you should not rely on this information for purchasing or planning purposes. Just like all projects, the items in the roadmap are subject to change or delay, and the continued development or release of a feature on the roadmap is at the sole discretion of the librpma development team.
+The roadmap is 
+
+Status|Title|Description|Expected
+---|---|---|---
+[x]|Connection|Be able to establish RDMA connection with a remote node|ww26'20
+[x]|Connection example|Example that demonstrates how to establish connection using librpma.|ww26'20
+[x]|Remote read|Be able to read remote persistent memory|ww28'20
+[x]|Read example|Example that demonstrates how to read remote persistent memory to local persistent memory.|ww28'20
+[x]|Connection data|Be able to exchange information during connection establish procedure|ww28'20
+[x]|Read example extended|An example that demonstrates how to use connection time data exchange do deliver a memory handle to the opposite side of a connection|ww28'20
+[ ]|Remote write|Be able to write to remote persistent memory|ww30'20
+[ ]|Remote flush|Be able to ensure data securely stored in a remote node|ww30'20
```

Comment:
I would be careful with the word "securely" - this might imply encryption or other security things